check-log/lib/check-log.go

```diff
@@ -22,21 +22,21 @@ import (
 )
 
 type logOpts struct {
-	LogFile             string  `short:"f" long:"file" value-name:"FILE" description:"Path to log file"`
-	Pattern             string  `short:"p" long:"pattern" required:"true" value-name:"PAT" description:"Pattern to search for"`
-	Exclude             string  `short:"E" long:"exclude" value-name:"PAT" description:"Pattern to exclude from matching"`
-	WarnOver            int64   `short:"w" long:"warning-over" description:"Trigger a warning if matched lines is over a number"`
-	CritOver            int64   `short:"c" long:"critical-over" description:"Trigger a critical if matched lines is over a number"`
-	WarnLevel           float64 `long:"warning-level" value-name:"N" description:"Warning level if pattern has a group"`
-	CritLevel           float64 `long:"critical-level" value-name:"N" description:"Critical level if pattern has a group"`
-	ReturnContent       bool    `short:"r" long:"return" description:"Return matched line"`
-	FilePattern         string  `short:"F" long:"file-pattern" value-name:"FILE" description:"Check a pattern of files, instead of one file"`
-	CaseInsensitive     bool    `short:"i" long:"icase" description:"Run a case insensitive match"`
-	StateDir            string  `short:"s" long:"state-dir" value-name:"DIR" description:"Dir to keep state files under"`
-	NoState             bool    `long:"no-state" description:"Don't use state file and read whole logs"`
-	Encoding            string  `long:"encoding" description:"Encoding of log file"`
-	Missing             string  `long:"missing" default:"UNKNOWN" value-name:"(CRITICAL|WARNING|OK|UNKNOWN)" description:"Exit status when log files missing"`
-	patternReg          *regexp.Regexp
+	LogFile             string   `short:"f" long:"file" value-name:"FILE" description:"Path to log file"`
+	Pattern             []string `short:"p" long:"pattern" required:"true" value-name:"PAT" description:"Pattern to search for. If specified multiple, they are treated as AND condition"`
```

Comment:
> If specified multiple, they are treated as AND condition

[nit][english]
How about something like,
`If specified multiple, they will be treated together with the AND operator.`